pytorch_toolkit/action_recognition_2/model_templates/gesture_recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -0,0 +1,69 @@
+name: s3d-rgb-mobilenet-v3-stream-jester
+domain: Action Recognition 2
+problem: Gesture Recognition
+framework: OTEAction
+summary: ASL Recognition based on S3D-MobileNet-V3.
+annotation_format: CustomAction
+classes: []
+dependencies:
+- sha256: d55d4a208e2a0d379026665d6d5bf7c191b9cc4c52672733df1df878a6b3d7c6
+  size: 38499945
+  source: https://download.01.org/opencv/openvino_training_extensions/models/asl/s3d-mobilenetv3-large-statt-kinetics700.pth
+  destination: imagenet1000-kinetics700-snapshot.pth
+- sha256: 3bb05087cb405f83479a5ae83ec6a8ad49163047569e0309f78f81e9a337708c
+  size: 43195590
+  source: https://download.01.org/opencv/openvino_training_extensions/models/asl/s3d-mobilenetv3-large-statt-msasl1000.pth
+  destination: snapshot.pth
+- source: ../../../../../pytorch_toolkit/ote/tools/train.py
+  destination: train.py
+- source: ../../../../../pytorch_toolkit/ote/tools/eval.py
+  destination: eval.py
+- source: ../../../../../pytorch_toolkit/ote/tools/export.py
+  destination: export.py
+- source: ../../../../../pytorch_toolkit/ote/tools/quantize.py
+  destination: quantize.py
+- source: ../../../../../pytorch_toolkit/ote
+  destination: packages/ote
+- source: ../../requirements.txt
+  destination: requirements.txt
+max_nodes: 1
+training_target:
+- GPU
+inference_target:
+- CPU
+- iGPU
+- VPU
+hyper_parameters:
+  basic:
+    batch_size: 14
+    base_learning_rate: 0.01
+    epochs: 65
+output_format:
+  onnx:
+    default: true
+  openvino:
+    default: true
+    input_format: RGB
+quantization: TBD
+metrics:
+- display_name: Size
+  key: size
+  unit: Mp
+  value: 4.133
+- display_name: Complexity
+  key: complexity
+  unit: GFLOPs
+  value: 6.66
+- display_name: Top-1 accuracy
+  key: accuracy
+  unit: '%'
+  value: 84.7
+gpu_num: 2
+tensorboard: true
+config: model.py
+modules:
```

Comment:
I don't like an idea when implementation details are presented in template.yaml file. Probably separate *.py module can be added to template.yaml folder so that after copying train.py/eval.py... this module will be loaded during running of corresponding script.

Reply:
Sounds good.

Reply:
Modules definition have been moved to `modules.yaml` file and all work to process it performs automatically during running one of `train`, `eval`, `export` scripts.